201-servicebus-create-topic/azuredeploy.parameters.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "serviceBusNamespaceName": {
+      "value": "GEN-UNIQUE"
+    },
+    "serviceBusTopicName": {
+      "value": "GEN-UNIQUE"
+    },
+    "serviceBusSku": {
```

Comment:
remove this and use the defaultValue in the template

Reply:
resolved